Consumer/src/main/java/com/consumer/client/ClientConsumer.java

```diff
@@ -0,0 +1,24 @@
+package com.consumer.client;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class ClientConsumer {
+
+  private final List<Client> clients = new ArrayList<>();
```

Comment:
Hmm, it's could be used only as an example :D (not thread safety, single instance only). original Training task includes insertions into Database.

Reply:
Yep, will implement it in separate PR as additional feature